src/js_modules/profile/Subscriptions/index.jsx

```diff
@@ -64,6 +65,20 @@ function Subscriptions({ cohorts }) {
 
   const getConsumables = async () => {
     try {
+      const nonSaasCohorts = cohorts.filter(({ available_as_saas }) => !available_as_saas);
+
+      const allServices = {
+        nonSaasMentorships: [],
+        mentorships: [],
+        workshops: [],
+      };
+
+      const cohortsServices = nonSaasCohorts.map(({ academy }) => bc.mentorship({ academy: academy.id }, true).getService());
```

Comment:
Shouldn't you reduce this nonSaasCohorts array to avoid duplicate academies and avoid repeating the request?